components/gfx/font_cache_task.rs

```diff
@@ -309,11 +322,11 @@ impl FontCacheTask {
         }
     }
 
-    pub fn find_font_template(&self, family: String, desc: FontTemplateDescriptor)
+    pub fn find_font_template(&self, family: &FontFamily, desc: FontTemplateDescriptor)
                                                 -> Option<Arc<FontTemplateData>> {
 
         let (response_chan, response_port) = ipc::channel().unwrap();
-        self.chan.send(Command::GetFontTemplate(family, desc, response_chan)).unwrap();
+        self.chan.send(Command::GetFontTemplate(family.name().to_owned(), desc, response_chan)).unwrap();
```

Comment:
Since this converts the family to a string here, I don't see how it can differentiate between the enum value and 'serif' as a string when selecting which font to return?